dipy/align/tests/test_vector_fields.py

```diff
@@ -798,13 +798,9 @@ def test_affine_transforms_2d():
     shape = np.array(codomain_shape, dtype=np.int32)
     # Exceptions from transform_2d
     assert_raises(ValueError, vfu.transform_2d_affine, circle, shape, invalid)
-    assert_raises(ValueError, vfu.transform_2d_affine, circle, shape, invalid)
-    assert_raises(ValueError, vfu.transform_2d_affine, circle, shape, invalid)
```

Comment:
Yes... in Notepad++, Ctrl+d means "duplicate current line" and 'd' is just besides 's'... I guess I accidentally pressed 'd' (twice?! =S) attempting to save...